contracts/common/lib/RLPEncode.sol

```diff
@@ -10,9 +10,9 @@ import "./BytesLib.sol";
 library RLPEncode {
 
   // Encode an item (bytes memory)
-  function encodeItem(bytes memory self) internal pure returns (bytes memory) {
+  function encodeItem(bytes memory self) public pure returns (bytes memory) {
     bytes memory encoded;
-    if (self.length == 1 && uint(bytes32(self[0] & 0xFF)) < 0x80) {
+    if (self.length == 1 && uint8(self[0] & 0xFF) < 0x80) {
```

Comment:
What comment should we add here, if any

Reply:
not sure tho, for now, we can add one `TODO: evaluate consequences`.

Reply:
that's why todo: 😛